src/components/DatePicker/index.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component, Fragment } from 'react';
+import { DatePicker } from 'antd';
+import { connect } from 'dva';
+import moment from 'moment';
+import styles from './index.less';
+
+const { RangePicker } = DatePicker;
+
+@connect(({ global, datastore }) => ({
+  indices: datastore.indices,
+  selectedDateRange: global.selectedDateRange,
+}))
+class AntdDatePicker extends Component {
+  handleChange = (dates, dateStrings) => {
+    const { dispatch, onChangeCallback } = this.props;
+    dispatch({
+      type: 'global/updateSelectedDateRange',
+      payload: dateStrings,
+    }).then(() => {
+      if (onChangeCallback != null) {
```

Comment:
What if we made `onChangeCallback` a required prop as opposed to checking whether it exists before invocation?